onnx_tf/handlers/backend/unique.py

```diff
@@ -0,0 +1,95 @@
+import tensorflow as tf
+import numpy as np
+
+from onnx_tf.handlers.backend_handler import BackendHandler
+from onnx_tf.handlers.handler import onnx_op
+from onnx_tf.handlers.handler import tf_func
+from onnx_tf.common import exception
+
+tf.config.run_functions_eagerly(True)
+
+
+@onnx_op("Unique")
+@tf_func(tf.unique_with_counts)
+@tf_func(tf.sort)
```

Comment:
These are not necessary since the tf functions are called directly, not via make_tensor_from_onnx_node method. Plus, each handler can map to only one tf function. Please check handler.py and backend_handler.py for details.

Reply:
addressed.